src/routes/slack/user.ts

```diff
@@ -101,6 +105,15 @@ async function getUserInfo(req, res, query) {
     attachments.push(getSlackAttachmentForMap('CivicLiker Info', userInfo.civicLiker));
     delete userInfo.civicLiker;
   }
+  if (userInfo.likerPlus) {
+    attachments.push(getSlackAttachmentForMap('Liker Plus Info', userInfo.likerPlus));
+    delete userInfo.likerPlus;
+  }
+  if (userInfo.bookInfo) {
+    attachments.push(getSlackAttachmentForMap('LikerNFT Book Info', userInfo.bookInfo));
```

Comment:
Book Info seems like a book to me
```suggestion
 attachments.push(getSlackAttachmentForMap('LikerNFT Book User Info', userInfo.bookInfo));
```

Reply:
Changed to `Book Press User Info`